spec/acceptance/nodesets/docker/ubuntu-14.04.yml

```diff
@@ -0,0 +1,15 @@
+HOSTS:
+  ubuntu-1404-x64:
+    platform: ubuntu-14.04-amd64
+    hypervisor : docker
+    image: ubuntu:14.04
+    docker_preserve_image: true
+    docker_cmd: '["/sbin/init"]'
+    docker_image_commands:
+      - 'rm /usr/sbin/policy-rc.d'
+      - 'rm /sbin/initctl; dpkg-divert --rename --remove /sbin/initctl'
```

Comment:
hrm… we should perhaps add .overrides for /etc/init/tty[1-6].conf to disable them so we don't leak them, causing travis to overload.